src/charm.py

```diff
@@ -744,20 +770,25 @@ def _stop_server(self):
         if self._wordpress_service_exists():
             self._container().stop(self._SERVICE_NAME)
 
-    def _wp_is_installed(self):
-        """Check if WordPress is installed (check if WordPress related tables exist in database)"""
-        logger.debug("Check if WordPress is installed")
+    def _run_wp_cli(self, cmd, timeout=60, combine_stderr=False):
+        Result = collections.namedtuple("WordpressCliExecResult", "return_code stdout stderr")
```

Comment:
Should this be `Result = collections.namedtuple("WordpressCliExecResult", ["return_code", "stdout stderr"])`?

Reply:
It works in both way https://docs.python.org/3/library/collections.html#collections.namedtuple
```python3
import collections

collections.namedtuple("Test", "a b")(a=1, b=2)
> Test(a=1, b=2)
```